libcxx/test/std/utilities/utility/mem.res/mem.res.monotonic.buffer/mem.res.monotonic.buffer.mem/allocate_growth_capacity.cpp

```diff
@@ -20,38 +20,40 @@
 #include "count_new.h"
 #include "test_macros.h"
 
-void test_geometric_progression() {
+// LWG 3143: https://cplusplus.github.io/LWG/issue3143
+void test_growth_capacity() {
   // mem.res.monotonic.buffer 1.3
-  // Each additional buffer is larger than the previous one, following a
-  // geometric progression.
+  // Each additional buffer is larger than the previous one
```

Comment:
Would it make sense to still test that libc++ does geometric progression? Perhaps as a libc++ specific test?

Reply:
IMO this test is OK as it is now, but we're missing an additional test for the libc++ specific behavior with the geometric progression -- unless I misunderstood what this LWG issue is about.

Reply:
I will add more test to test the behavior